### PR TITLE
Update InGameInfo.xml - Adding center chunk icon & removing unecessaey chunk checks

### PR DESCRIPTION
**Pull request by PlayfulPiano**
Noticed that the current .xml file has an unnecessary chunk coordinate check (chunk coordinates, not relative coordinates within a chunk) for the "center of the chunk" line. I removed that part and added in the obsidian tough binding icon to be placed next to this line.

I would've added some sort of red tough binding icon but I could not figure out the proper file path for like an ignis tough binding. So obsidian it is.

Also uhhh ignore that second commit. I tried to fix one of the lines using a different tab size but it literally did nothing and could still be saved as a commit so /shrug

### CODE REVIEW

**Comment by PlayfulPiano** [comment]
trying to put this into draft, now realizing that the PR https://github.com/GTNewHorizons/InGame-Info-XML/pull/24 seems to have caused a lot of issues with icons displaying. can't get this pushed until that first is fixed. (this code worked fine in beta 2 but beta 3 seems to have multiple icons broken)

**Comment by PlayfulPiano** [comment]
This can be undrafted as of the latest igixml version. Technically a day after deadline but the PR was made before so /shrug

File: config/InGameInfoXML/InGameInfo.xml [hunk 1]
```diff
@@ -1039,44 +1039,10 @@
...
-                                        <num>3</num>
-                                    </modi>
-                                    <num>1</num>
-                                </equal>
-                            </or>
-                        </and>
-                        <str>      </str>
-                    </if>
-                    <str>   You are at the {darkred}center {white}of the chunk.</str>
+					<icon>
```
Comment (wlhlm) [inline config/InGameInfoXML/InGameInfo.xml:1042]: Please use spaces instead of tabs for indendation.

**Review by wlhlm — CHANGES_REQUESTED** [review]
Looks good other than the minor formatting nit.

File: config/InGameInfoXML/InGameInfo.xml (hunk 1 above)
Comment (PlayfulPiano) [reply config/InGameInfoXML/InGameInfo.xml]: done, new commit has the changes to indentation

**Review by wlhlm — APPROVED** [review]

**Pull request merged**